src/block_cache/block_chunkcache.c

```diff
@@ -368,6 +449,12 @@ __wt_chunkcache_get(WT_SESSION_IMPL *session, WT_BLOCK *block, uint32_t objectid
                 return (ret);
             }
 
+            /* Mark chunk as pinned if the chunk belongs to the object in pinned object array. */
+            WT_BINARY_SEARCH_STRING(chunk->hash_id.objectname, chunkcache->pinned_objects,
```

Comment:
Since @korteland is changing something related to object name, I will wait for that PR to be merged and incorporate the necessary changes.

Reply:
No need for that - we can merge this first if it's ready first.